[[[
Fix segfault when subversion is built with gnome keyring support and keyring
password lookup fails.

* subversion/libsvn_subr/simple_providers.c
  (svn_auth__simple_creds_cache_get): Initialize 'done' with FALSE since ie
    password_get_gnome_keyring sets 'done' only to TRUE. In case of error
    it leaves unintialized 'done' (usually non 0) and default_password with NULL
    causing segfault at subsequent strcmp.
]]]
Index: subversion/libsvn_subr/simple_providers.c
===================================================================
--- subversion/libsvn_subr/simple_providers.c	(revision 1835628)
+++ subversion/libsvn_subr/simple_providers.c	(working copy)
@@ -206,7 +206,7 @@
         {
           if (have_passtype)
             {
-              svn_boolean_t done;
+              svn_boolean_t done = FALSE;
 
               SVN_ERR(password_get(&done, &default_password, creds_hash,
                                    realmstring, username, parameters,
